specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Automated management of containerized applications.                                                                                                                                                    |
+|                           | End User Computing                | Desktop virtualization, mobile computing, and self-service analytics.                                                                                                                                  |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Facilitates building and running applications without managing servers.                                                                                                                                |
+|                           | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                                        |
+|                           | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                                                                                                               |
+| Databases                 | Caching                           | Frequently accessed data housed in a temporary storage area for quick retrieval.                                                                                                                       |
+|                           | Data Warehouse                    | Centralized repository that stores large volumes of structured data to facilitate analytical use cases.                                                                                                |
+|                           | Relational                        | Data organized into tables with rows and columns to facilitate transactional use cases.                                                                                                                |
```

Comment:
```suggestion
|                           | Relational                        | Storage and querying capabilities optimized for structured data.                                                                                                                |
```

Reply:
Not onboard with this one.  Don't want to use the word `storage` to describe something in the `Databases` category, given that we have a `Storage` category.

Reply:
```suggestion
|                           | Relational                        | Structured data storage and querying capabilities.                                                                                                                |
```